### PR TITLE
Exclude beebot from the credits contrib list

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

BeeBot will no longer show up in the contrib list in the credits.

## Changelog
:cl:
fix: BeeBot no longer shows up in the contrib credits
/:cl:

### CODE REVIEW

File: tools/fetchContributors.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 import argparse
 import operator
 
+# List of github usernames to excluse from the output file
+blacklist = [
+	"ss13-beebot"
```
Comment (Crossedfall) [inline tools/fetchContributors.py:11]:
```suggestion
	"ss13-beebot"
	"dependabot[bot]"
```

**Comment by qwertyquerty** [comment]
nice meme crossed

2d8b4cd864cd5d2f1a480d2cb62e5cab0d0d9400

**Comment by grimmmy** [comment]
Beebot is very essential to contributing, I am appalled you would do such a thing.

File: tools/fetchContributors.py [hunk 2]
```diff
@@ -6,6 +6,12 @@
 import argparse
 import operator
 
+# List of github usernames to excluse from the output file
```
Comment (ike709) [inline tools/fetchContributors.py:9]: > excluse

**Pull request merged**